index.html

```diff
@@ -1,231 +1,373 @@
-            </nav>
-            <div id="header-heading">
-                Hey there. I’m Clifford.
-                <span>I’m a software developer</span>
+      <!-- Mobile Header begins-->
+      <header id="mobile-header">
+        <nav class="top-nav pl-24 pr-24">
+          <p>Welcome</p>
+          <img
+            id="hamburger-icon"
+            src="./assets/images/hamburger-icon.svg"
+            alt="hamburger-icon"
+          />
+        </nav>
+        <div id="mobile-header-content">
+          <div id="mobile-header-heading" class="mr-24 ml-24">
+            <p id="hero-name" class="mb-38">
+              Hey there. I’m Clifford.<br />
+              <span>I’m a software developer</span>
+            </p>
+            <p id="hero-desc" class="mb-38">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+          <div id="mobile-header-social">
+            <ul>
+              <li><img src="../assets/images/github.svg" alt="github" /></li>
+              <li>
+                <img src="../assets/images/linkedin.svg" alt="linkedin" />
+              </li>
+              <li>
+                <img src="../assets/images/angelist.svg" alt="angelist" />
+              </li>
+              <li><img src="../assets/images/twitter.svg" alt="twitter" /></li>
+              <li><img src="../assets/images/medium.svg" alt="medium" /></li>
+            </ul>
+          </div>
+          <img
+            class="arrow-down"
+            src="../assets/images/arrow-down.svg"
+            alt="arrow-down"
+          />
+        </div>
+      </header>
+      <!-- Mobile header ends -->
+      <header id="desktop-header">
+        <nav class="top-nav">
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <div id="header-heading">
+          Hey there. I’m Clifford.
+          <span>I’m a software developer</span>
+        </div>
+        <div id="header-desc">
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div id="header-social">
+          <ul>
+            <li><img src="../assets/images/github.svg" alt="github" /></li>
+            <li><img src="../assets/images/linkedin.svg" alt="linkedin" /></li>
+            <li><img src="../assets/images/angelist.svg" alt="angelist" /></li>
+            <li><img src="../assets/images/twitter.svg" alt="twitter" /></li>
+            <li><img src="../assets/images/medium.svg" alt="medium" /></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Please work on these icons and make them visible

-
![a](https://user-images.githubusercontent.com/63972670/173920458-fccf0efa-6c79-4a00-8efd-99ec047de086.PNG)